packages/preview-middleware-client/test/unit/adp/controllers/AddFragment.controller.test.ts

```diff
@@ -11,6 +11,9 @@ import ControlUtils from '../../../../src/adp/control-utils';
 import AddFragment from '../../../../src/adp/controllers/AddFragment.controller';
 import rtaMock from 'mock/sap/ui/rta/RuntimeAuthoring';
 import { ValueState } from 'mock/sap/ui/core/library';
+import OverlayRegistry from 'mock/sap/ui/dt/OverlayRegistry';
+import Popover from 'sap/m/Popover';
+import ManagedObject from 'sap/ui/base/ManagedObject';
```

Comment:
Change to import type for Popover and ManagedObject

Reply:
resolved in [e94a899](https://github.com/SAP/open-ux-tools/pull/1477/commits/e94a89919304cadb80bf98ae1eb4ece1069cc8cb)

Reply:
Still need to import type ManagedObject

Reply:
[37cc141](https://github.com/SAP/open-ux-tools/pull/1477/commits/37cc141c0f4e040454ed96e66536e4c64f27436b)